sunpy/map/tests/test_mapbase.py

```diff
@@ -794,12 +794,6 @@ def test_as_mpl_axes_aia171(aia171_test_map):
     assert all([ct1 == ct2 for ct1, ct2 in zip(ax.wcs.wcs.ctype, aia171_test_map.wcs.wcs.ctype)])
 
 
-def test_pixel_to_world_no_projection(generic_map):
-    out = generic_map.pixel_to_world(*u.Quantity(generic_map.reference_pixel))
-    assert_quantity_allclose(out.Tx, 0*u.arcsec)
-    assert_quantity_allclose(out.Ty, 0*u.arcsec)
```

Comment:
@wtbarnes why this gone?

Reply:
it was testing the lack of projection, but `generic_map` has a projection now.

Although, that being said, odd it was failing at all... (even if the name is irrelevant now)

Reply:
I can revert it if we want to keep it in.

Reply:
Cool, just wanted to double check, am happy with it gone 👍